pennylane_orquestra/qiskit_device.py

```diff
@@ -46,22 +42,20 @@ class QeQiskitDevice(OrquestraDevice):
     qe_module_name = "qeqiskit.simulator"
     qe_function_name = "QiskitSimulator"
 
-    def __init__(self, wires, shots=10000, backend="qasm_simulator", **kwargs):
+    def __init__(self, wires, shots=1000, backend="qasm_simulator", **kwargs):
```

Comment:
How come we have it lowered here, though for Forest we still have 10k?

Reply:
Oh, I don't know, maybe it was an accident?